crates/bevy_asset/src/processor/mod.rs

```diff
@@ -61,6 +83,7 @@ pub struct AssetProcessor {
     pub(crate) data: Arc<AssetProcessorData>,
 }
 
+/// The data stored inside an [`AssetProcessor`].
```

Comment:
Too generic. Please give some idea of what said data is used for. All structs contain data, usually :P

Reply:
Agreed. It seems to be just a "here's a blob of data wrapped in an Arc" with not a ton of rhyme or reason, and with very few publicly useful methods. I tried making it private but it is used in a few places, both in the crate and publicly.

Reply:
You can just say "Internal data used by an [AssetProcessorThing]" then imo.